tripl/tripl.py

```diff
@@ -72,7 +76,7 @@ def __iter__(self):
                 for v in vs:
                     yield k, v
             else:
-                for tupl in v:
+                for tupl in vs:
```

Comment:
Did this find a bug?

Reply:
PyCharm said `v` might no be assigned before use. `vs` seemed to be the closest to what might have been meant.

Reply:
I think you're right that this is the only thing that seems to make sense. Will look into it further. Thanks again!